builds.json

```diff
@@ -53,7 +53,7 @@
         {
           "package": "org.gel.models.cva.avro",
           "python_package": "cva",
-          "version": "1.1.0",
+          "version": "1.2.0",
```

Comment:
There is no need for new version of cva `1.2.0`. Current `1.1.0` is not used and it can be completely replaced.